src/firebase-config.js

```diff
@@ -0,0 +1,19 @@
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+import { getFirestore } from "firebase/firestore";
+
+const firebaseConfig = {
+  apiKey: "AIzaSyBBIMXMWrWCPlzialR25ctSEFIBf7KGgGo",
+  authDomain: "stint-pomodoro.firebaseapp.com",
+  projectId: "stint-pomodoro",
+  storageBucket: "stint-pomodoro.appspot.com",
+  messagingSenderId: "139209721745",
+  appId: "1:139209721745:web:64abe29a1d169cd02cfc30",
+  measurementId: "G-GFMNWSEWHZ",
+};
```

Comment:
Do not expose your firebase credentials. Keep it in .env file

Reply:
Done moved to env and incorporated env variables in config